python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2

```diff
@@ -75,7 +75,7 @@ dot1x
    mac based authentication hold period {{ dot1x.mac_based_authentication.hold_period }} seconds
 {%             endif %}
 {%         endif %}
-{%         if dot1x.radius_av_pair is arista.avd.defined %}
+{%         if dot1x.radius_av_pair.service_type is arista.avd.defined(true) or dot1x.radius_av_pair.framed_mtu is arista.avd.defined %}
```

Comment:
I think this if-else statement can be removed.

Reply:
updated